<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortDisplayer</class>
 <widget class="QMainWindow" name="SerialPortDisplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialPortDisplayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="VLay_Main">
    <item>
     <widget class="QTableView" name="TableView_Ports"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="HLay_Buttons">
      <item>
       <widget class="QPushButton" name="PshBtn_Refresh">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PshBtn_OpenSelected">
        <property name="text">
         <string>Open selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PshBtn_ReadOpened">
        <property name="text">
         <string>Read opened</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="HSpacer_Buttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="MainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
